src/source/Ice/IceAgent.c

```diff
@@ -2604,17 +2604,30 @@ STATUS iceAgentFatalError(PIceAgent pIceAgent, STATUS errorStatus)
 VOID iceAgentLogNewCandidate(PIceCandidate pIceCandidate)
 {
     CHAR ipAddr[KVS_IP_ADDRESS_STRING_BUFFER_LEN];
-    PCHAR protocol = "UDP";
+    PCHAR protocol = "UNKNOWN";
```

Comment:
Should we maintain this to be NONE instead of UNKNOWN to align with the KVS_SOCKET_PROTOCOL_* enum?

Reply:
Yeah I do that below in the switch.  This is for the case that it was none of those values, suppose we add a new enum or pIceCandidate was NULL or pIceCandidate->iceCandidateType ==!= ICE_CANDIDATE_TYPE_RELAYED

Reply:
We should be consistent. There is a usage of 3 terms in this function - NA, NONE and UNKNOWN. Please chose one. I would gravitate towards UNKNOWN

Reply:
They all have different meaning.

* NA means the turn connection was NULL (which makes sense that if turn connection is none, protocol is really NA)
* NONE is just propagating the already set protocol filed in the turn connection
* UNKNOWN specifically means that it was none of those other cases so we really do not know.

Reply:
The whole point of debugging messages is to be as specific as possible, with the 3 distinct values we'll know more about the state of the variables as opposed to mashing them all together into one.

Reply:
Oh OK. I had approved it